bang/models.py

```diff
@@ -883,7 +883,8 @@ def t_stamp_translation(self):
         'i_rarity': 3,
     }, on_delete=models.SET_NULL)
     secondary_card = models.ForeignKey(Card, related_name='secondary_card_event', null=True, limit_choices_to={
-        'i_rarity': 2,
```

Comment:
You can also do:

```python
{
    'i_rarity__in': [2, 3],
}
```

That would be more readable imo. Not sure which SQL query would be more performant between the 2 though.

Reply:
Performance shouldn't be an issue since that query is only made from the staff form afaik, so I've switched to using `in` (on the main card limit clause as well, for consistency).

I also made the form take allowed rarities from the model so we're not defining the same constant in multiple places.